components/aws/sagemaker/requirements_v2.txt

```diff
@@ -2,4 +2,5 @@
 pathlib2==2.3.5
 pyyaml==5.4.1
 mypy-extensions==0.4.3
-kubernetes==12.0.1
\ No newline at end of file
+kubernetes==12.0.1
+urllib3==1.26.15
```

Comment:
where is urllib3 being used?

Reply:
We set the version to be the latest V1.0 version, because urllib3.0 v2.0 (released days ago) only supports OpenSSL 1.1.1+, but amazonlinux is using penssl 1:1.0.2k-24.amzn2.0.6.